tests/components/workday/test_config_flow.py

```diff
@@ -653,3 +653,48 @@ async def test_form_with_categories(hass: HomeAssistant) -> None:
         "language": "de",
         "category": ["half_day"],
     }
+
+
+async def test_options_form_removes_subdiv(hass: HomeAssistant) -> None:
+    """Test we get the form in options when removing a configured subdivision."""
+
+    entry = await init_integration(
+        hass,
+        {
+            "name": "Workday Sensor",
+            "country": "DE",
+            "excludes": ["sat", "sun", "holiday"],
+            "days_offset": 0,
+            "workdays": ["mon", "tue", "wed", "thu", "fri"],
+            "add_holidays": [],
+            "remove_holidays": [],
+            "language": "de",
+            "province": "BW",
+        },
+    )
+
+    result = await hass.config_entries.options.async_init(entry.entry_id)
```

Comment:
better variable naming would be nice - but this file follows pattern of unspecified variables.